src/hook/mypage/useEditNicknameMutation.ts

```
@@ -0,0 +1,21 @@
+import api from '@hook/api.ts';
+
+interface NicknameProps {
+  newNickName: string;
+}
+export const useEditNicknameMutation = () => {
+  return useMutation({
+    mutationFn: (payload: NicknameProps) =>
+      api.put('/v1/member/nickname', payload),
+    onSuccess: (response) => {
+      if (response.data.success) {
+      } else {
```

ESLint

[error] Empty block statement.

(no-empty)